blazar/utils/trusts.py

```diff
@@ -35,9 +35,10 @@ def create_trust():
     ctx = context.current()
     trust = client.trusts.create(trustor_user=ctx.user_id,
                                  trustee_user=trustee_id,
-                                 impersonation=False,
+                                 impersonation=True,
                                  role_names=ctx.roles,
-                                 project=ctx.project_id)
+                                 project=ctx.project_id,
+                                 allow_redelegation=True)
```

Comment:
Did we determine if redelegation was actually necessary? If not, let's remove it.

Note for @priteau: `impersonation` looks necessary due to how the ACLs work on Nova Keypairs - it only allows a token to list keypairs if the token's user matches the owner of the keypair. I'm not sure why this is and it feels like something that could/should be fixed in Nova, but it feels like there are likely other places that have not been updated to support Keystone Trusts and rely on the token's user to match the resource.

Reply:
Didn't effect launching a single instance via heat.